ols-2.md

```diff
@@ -169,46 +194,52 @@ During these calls:
 - Speakers will present their work related to the topic of the week
 - Participants will be given group discussion exercises
 - An open Q&A will be run and notes will be co-developed
-
-  Look for cohort notes shared with you by organisers
-
 - Exercises will be given for the week to be completed before the mentee-mentor meeting
 
-The calls will be hosted online using the Zoom web-conferencing option. A link for the calls will be shared for each meeting separately.
+The calls will be hosted online using the Zoom web-conferencing option.
+A link for the calls will be shared for each meeting separately.
+
+Look up the shared notes for each call linked to the [schedule](#schedule) in this website.
+You will also be updated via email each week by the organisers with additional details to aid your participation.
 
 **If you can't make it to a call**:
-- If you have to miss a call, please write your name in the cohort note under "apologies".
 
-  The call will be recorded and available on the [OLS YouTube channel]({{ site.youtube }}) after the call.
+The call will be recorded and available on the [OLS YouTube channel]({{ site.youtube }}) after the call.
 
-- If you miss two or more calls during the program, we'll evaluate if you would be able to finish the program.
+If you can not attend most calls during the program due to the time zone incompatibility or other personal obligation, please let the organisers know.
+If you miss to communicate with your mentor regularly or not engage in the program as planned, we'll evaluate if you would be able to finish the program.
 
 ## Mentor-mentee calls
 
 The Mentor-mentee calls take place **every 2 weeks** (unless mentioned otherwise) and last for **30 minutes**.
 
 During these calls:
 - Mentors help their mentees evaluate their understanding of the new topics
-- Mentees will complete their task assigned at the cohort calls using new skills learned that week 
+- Mentees will complete their task assigned at the cohort calls using new skills learned that week
 - Mentors and mentee will review progress together where mentees provide constructive feedback
+- Mentors will connect mentees with other experts and get consulted on their project when needed
 
-  Look for 1:1 notes shared with you by your mentor
+Coordinate with your mentor how you manage the notes and assignments for your 1:1 calls.
 
-- Mentors will connect mentees with other experts and get consulted on their project when needed
+The online communication options can be agreed upon by the mentor-mentee pairs.
+A few options to explore are the following:
 
-The calls can use the online communication options that both mentor and mentee agree to use. A few options to explore are the following:
 - Zoom: 40 mins limit for each call
 - Google hangout: Free for members with google account
 - Skype: Free, download the app
 - Whereby.com: Free option, valid upto 4 participants
+- Jitsy: Free, web-based call is possible
```

Comment:
```suggestion
- [Jitsi](https://meet.jit.si/): Free, open source web-based call is possible
```

Reply:
typo fix + added link while I was there :)

Reply:
Added the change in a different PR